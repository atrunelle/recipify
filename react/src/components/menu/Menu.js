import React, { Component } from 'react';
import Toolbar from '@material-ui/core/Toolbar'; 
import './Menu.css';
import { Link } from 'react-router-dom';

class Menu extends Component {
  render() {
    return (
      <Toolbar>
        <nav>
          <Link to="/">
              Create recipe
          </Link>
          <Link to="/recipes/list">
            My recipes
          </Link>
        </nav>
      </Toolbar>
    );
  }
}

export default Menu;