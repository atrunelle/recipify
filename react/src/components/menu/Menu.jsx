import React, { Component } from 'react';
import './Menu.css';
import { Link } from 'react-router-dom';
import { Button, Toolbar, Grid } from '@material-ui/core';

class Menu extends Component {
  render() {
    return (
      <Toolbar>
        <Grid
        container
        justify="flex-end">
          <nav>
            <Button 
              component={Link}
              to="/">
              Create recipe
            </Button>
            <Button
              component={Link}
              to="/recipes/list">
              My recipes
            </Button>
          </nav>
        </Grid>
      </Toolbar>
    );
  }
}

export default Menu;