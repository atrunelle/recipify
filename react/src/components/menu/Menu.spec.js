import React from 'react';
import { mount } from 'enzyme';
import createRouterContext from 'react-router-test-context';
import PropTypes from 'prop-types';
import Menu from './Menu';

it('should match snapshot', () => {
  const context = createRouterContext()
  const childContextTypes = {
    router: PropTypes.object,
  };
  
  describe('Component: Menu', () => {
    const component = mount(
      <Menu />,
      { context, childContextTypes },
    );
    expect(component.html()).toMatchSnapshot();
  });
});