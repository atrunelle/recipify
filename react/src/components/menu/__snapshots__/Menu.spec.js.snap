// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `"<div class=\\"MuiToolbar-root-1 MuiToolbar-gutters-2\\"><div class=\\"MuiGrid-container-3 MuiGrid-justify-xs-flex-end-21\\"><nav><a tabindex=\\"0\\" class=\\"MuiButtonBase-root-110 MuiButton-root-94\\" role=\\"button\\" href=\\"/\\"><span class=\\"MuiButton-label-95\\">Create recipe</span><span class=\\"MuiTouchRipple-root-113\\"></span></a><a tabindex=\\"0\\" class=\\"MuiButtonBase-root-110 MuiButton-root-94\\" role=\\"button\\" href=\\"/recipes/list\\"><span class=\\"MuiButton-label-95\\">My recipes</span><span class=\\"MuiTouchRipple-root-113\\"></span></a></nav></div></div>"`;
